rfcs/experimental/flow_graph_node_sets/README.md

```diff
@@ -0,0 +1,262 @@
+# Helper Functions for Expressing Flow Graphs
+
+## Introduction
+
+The flow graph API lacks a simple way to create multiple edges with a single call. The result of this
+limitation is verbose, less-readable code.  For example, the following picture shows a flow graph where the
+node `input` has three successors.
+
+<img src="graph_with_many_edges.png">
+
+Without the proposed extensions, three separate calls to `make_edge` are required to connect the
+single `input` node to its three successors: 
+
+```cpp
+    broadcast_node<int> input(g);
+
+    function_node doubler(g, unlimited, [](const int& v) { return 2 * v; });
+    function_node squarer(g, unlimited, [](const int&) { return v * v; });
+    function_node cuber(g, unlimited, [](const int& v) { return v * v * v; });
+
+    make_edge(input, doubler);
+    make_edge(input, squarer);
+    make_edge(input, cuber);
+```
+
+To reduce verbosity and improve readability, additional functions were added to the flow graph API
+to simpify these cases. A shorter and more readable implementation of the previous code snippet that
+uses these extensions is shown below:
+
+```cpp
+    function_node doubler(g, unlimited, [](const int& v) { return 2 * v; });
+    function_node squarer(g, unlimited, [](const int&) { return v * v; });
+    function_node cuber(g, unlimited, [](const int& v) { return v * v * v; });
+
+    broadcast_node input(precedes(doubler, squarer, cuber));
+```
+
+## Proposal
+
+Several helper functions and additional constructors have been added as experimental features.
+These additions simplify expressing dependencies between nodes when building oneTBB flow graphs.
+This experimental feature is described in more detail in the
+[Developer Guide](https://www.intel.com/content/www/us/en/docs/onetbb/developer-guide-api-reference/2022-0/helper-functions-for-expressing-graphs.html)
```

Comment:
Technically, it's in the API Reference document, not in the Developer Guide.
Also, would it be better to refer to github.io pages?https://uxlfoundation.github.io/oneTBB/main/reference/helpers_for_expressing_graphs.html